pkgs/applications/terminal-emulators/kitty/update.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl gnugrep gnused jq
+
+set -x -eu -o pipefail
+
+NIXPKGS_PATH="$(git rev-parse --show-toplevel)"
+KITTY_PATH="$(
+  cd -- "$(dirname "$0")" >/dev/null 2>&1
+  pwd -P
+)"
+
+OLD_VERSION="$(nix-instantiate --eval -E "with import $NIXPKGS_PATH {}; kitty.version or (builtins.parseDrvName kitty.name).version" | tr -d '"')"
+LATEST_TAG=$(curl ${GITHUB_TOKEN:+" -u \":$GITHUB_TOKEN\""} --silent https://api.github.com/repos/kovidgoyal/kitty/releases/latest | jq -r '.tag_name')
+LATEST_VERSION=$(echo ${LATEST_TAG} | sed 's/^v//')
+
+if [ "$OLD_VERSION" = "$LATEST_VERSION" ]; then
+  echo "kitty is already up-to-date at $OLD_VERSION"
+  exit 0
+fi
+
+HASH=$(nix-prefetch-url --quiet --unpack https://github.com/kovidgoyal/kitty/archive/refs/tags/${LATEST_TAG}.tar.gz)
+HASH=$(nix hash to-sri --type sha256 "$HASH")
```

Comment:
```suggestion
HASH=$(nix --experimental-features=nix-command hash to-sri --type sha256 "$HASH")
```

Reply:
FYI it's `--experimental-features nix-command`. The = is invalid apparently.

Reply:
I forgot the CLI parser does not make any sense, thanks.